google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -35,6 +35,7 @@ appengine.promote.label=Promote the deployed version to receive all traffic
 appengine.promote.info.label={0}To manually promote a version use the {1}Google Cloud Console{2}{3}
 appengine.stop.previous.version.label=Stop previous version
 appengine.environment.label=Environment\:
+appengine.cloudsdk.devappserver.missing=Selected Cloud SDK does not contain the app-engine-java component.
```

Comment:
I thought of giving instructions on how to add the GAE SDK in my message, but thought that it would look a bit ugly in the warning text.
I'm thinking telling the user that the component is missing is a hint big enough to will let them find out they need to "gcloud components install". What do you think?